### PR TITLE
Reserve port 9273 for prometheus metrics

### PR DESCRIPTION
**Pull request by philipnrmn**
## High-level description

This PR reserves the port 9273 for dcos-metrics' Prometheus producer. Additionally, a check is added to ensure that this port is available on the node. 

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

 - [DCOS-20073](https://jira.mesosphere.com/browse/DCOS-20073) Reserve port 9273 for dcos-metrics

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS_OSS-2046](https://jira.mesosphere.com/browse/DCOS_OSS-2046) Add Prometheus Producer for Metrics

## Checklist for all PRs

  - [x] Included a test which will fail if code is reverted but test is not. 
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates (*not applicable*):

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by philipnrmn** [comment]
@bernadinm @amitaekbote would you kindly review this PR?

**Comment by philipnrmn** [comment]
@mesosphere-mergebot label ready for review

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by philipnrmn** [comment]
This PR required a rebase (I'm silly) and so I'm going to open a new enterprise PR.

**Comment by philipnrmn** [comment]
@mesosphere-mergebot bump-ee

**Comment by adamtheturtle** [comment]
Please make it clear between this and [the JIRA issue](https://jira.mesosphere.com/browse/DCOS-20073) that this does not close the JIRA issue. In particular, "Document this port. " is not covered by this PR.

**Comment by klueska** [comment]
The documentation work is being tracked in https://jira.mesosphere.com/browse/DCOS_OSS-2093

**Comment by jgehrcke** [comment]
@mesosphere-mergebot ship_it_label_status_check

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -515,7 +515,7 @@ package:
 {% endswitch %}
   - path: /etc/mesos-slave
     content: |
-      MESOS_RESOURCES=[{"name":"ports","type":"RANGES","ranges": {"range": [{"begin": 1025, "end": 2180},{"begin": 2182, "end": 3887},{"begin": 3889, "end": 5049},{"begin": 5052, "end": 8079},{"begin": 8082, "end": 8180},{"begin": 8182, "end": 32000}]}}]
+        MESOS_RESOURCES=[{"name":"ports","type":"RANGES","ranges": {"range": [{"begin": 1025, "end": 2180},{"begin": 2182, "end": 3887},{"begin": 3889, "end": 5049},{"begin": 5052, "end": 8079},{"begin": 8082, "end": 8180},{"begin": 8182, "end": 9272},{"begin": 9274, "end": 32000}]}}]
```
Comment (bbannier) [inline gen/dcos-config.yaml]: There seem to be too many spaces here.

Comment (jgehrcke) [reply gen/dcos-config.yaml]: Does that break something that is not covered by integration tests?

Comment (philipnrmn) [reply gen/dcos-config.yaml]: Thanks for catching this (curse you, autofmt!). I fixed it.

**Comment by branden** [comment]
@philipnrmn Could you add this port to https://github.com/dcos/dcos/blob/master/docs/admin-ports.md as well? This PR is mostly through CI so a follow-up PR is OK with me.

**Comment by amitaekbote** [comment]
@mesosphere-mergebot bump-ee

**Comment by amitaekbote** [comment]
@mesosphere-mergebot ship_it_label_status_check

**Pull request merged**